docs/blog/2022-11-21-epss.mdx

```diff
@@ -91,20 +103,27 @@ That's too bad for us, because one of the problems we are trying to solve with [
 is tuning our vulnerability prioritization to consider the environment the code is running in. We are working hard to build our own source
 of signal for that.
 
-### The result is opaque
-CVSS scores are calculated based on a limited set of inputs and the mathematical formula can be evaluated by a human.
+### The results are opaque
+
+CVSS scores are calculated based on a limited set of inputs and the mathematical formula can be manually computed with a calculator in just a few minutes.
+
+In contrast the opaque machine learning based approach of EPSS is very resistant to manual human calculation.
+The model is not Open Source and it is more difficult
+to introspect the results. The only output ever surfaced to consumers is the EPSS score itself, a value between 0 and 1.
+
+## Is EPSS relevant?
+
+As far as we can tell, EPSS is a good idea and is a much needed step that helps companies effectively prioritize their security efforts.
 
-In contrast, the machine learning based approach of EPSS is more opaque. The model is not open source, and it is more difficult
-to introspect the results. The only output of the model surfaced to consumers is the EPSS score itself. Previous versions
-of the model were detailed in academic papers
+But, while it's an exciting system that will eventually help solve the problem of too many CVEs being [false positives](https://www.lunasec.io/docs/blog/the-issue-with-vuln-scanners/), it's still early in it's maturity and very few security tools even include it today. There is also more validation required before security experts are willing to put their weight behind recommending that companies or standard use EPSS[^4].
 
-## Is EPSS relevant
+## How to try out EPSS
 
-The shortcomings of CVSS and existing scanning tools are pretty well known, and in fact, we are trying to build a business
-disrupting that. As far as we can tell, EPSS is a good idea and a needed step on the way to less bogus work for security engineers.
+While EPSS is only officially available as an API, we've been working to add EPSS support into the security tool we've been building on GitHub[^5][^6] that aggregates vulnerability data in one place.
 
-Once we get more of a feel for the data over the next month or so, we'd like to integrate showing EPSS scores into [LunaTrace](/pages/lunatrace/overview/introduction/).
-After all, it's free and would provide value to our users. Please let us know if this is something you're excited about.
+If you'd like to see a list of CVEs in your projects and the various risk scores available for each of them, you can do so for free today by [signing up for LunaTrace](https://lunatrace.lunasec.io/?ref=epss-blog) in just a few seconds. If you'd like participate in testing out our EPSS support, there are instructions after you login to LunaTrace for how to do so.
```

Comment:
This isn't true? Why don't we keep it as a true statement, it didn't hurt anyone.

Reply:
I was going to add in some text to the UI really quick with instructions for how to message us about becoming a beta user. (It'd be helpful to get people to explicitly ask so that we can figure out how to best shape this feature to their needs)